rpms/python-mongoengine/python-mongoengine.spec

```diff
@@ -0,0 +1,92 @@
+%define srcname mongoengine
+
+
+Name:           python-%{srcname}
+Version:        0.10.5
+Release:        2%{?dist}
+BuildArch:      noarch
+
+License:        MIT
+Group:          Development/Libraries
+Summary:        A Python Document-Object Mapper for working with MongoDB
+URL:            https://github.com/MongoEngine/mongoengine
+Source0:        https://github.com/MongoEngine/mongoengine/archive/v%{version}.tar.gz
+
+BuildRequires:  python-django
+BuildRequires:  python-pymongo
+BuildRequires:  python-pymongo-gridfs
+BuildRequires:  python-setuptools
+BuildRequires:  python-sphinx
+BuildRequires:  python2-devel
```

Comment:
Looking at the source (https://github.com/MongoEngine/mongoengine/tree/v0.10.5), I can't explain these build dependencies. Do you know if there is a reason for them, and if so, could it be documented?

Looking at https://github.com/MongoEngine/mongoengine/blob/2e406d26875c512548ed30a5d6760c555b44bffe/docs/changelog.rst#changes-in-0100 makes me think these requirements are no longer necessary... not that it's worth the time investigating build requirements for a dependency we don't plan to have for long.